hooks/conf_regen/15-nginx

```diff
@@ -18,9 +18,12 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 
+# Exit hook on subcommand error or unset variable
 set -e
```

Comment:
Test seems to fail for regenconf nginx, so may be that's why we keep only set -e ?